### PR TITLE
LPS-153063 Improve performance of Style Books

### PR DESCRIPTION
**Pull request by veroglez**

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by veroglez** [comment]
ci:test:sf

File: modules/apps/style-book/style-book-web/src/main/resources/META-INF/resources/js/style-book-editor/StyleBookEditor.js [hunk 1]
```diff
@@ -25,6 +25,28 @@ import {LayoutContextProvider} from './contexts/LayoutContext';
 import {StyleBookEditorContextProvider} from './contexts/StyleBookEditorContext';
 import {useCloseProductMenu} from './useCloseProductMenu';
 
+const getFrontendTokens = ({frontendTokenCategories}) => {
```
Comment (victorg1991) [inline modules/apps/style-book/style-book-web/src/main/resources/META-INF/resources/js/style-book-editor/StyleBookEditor.js:28]: Can we move this function to the botton of the file? We _usually_ prefer having the main component at the top of the file.

**Comment by victorg1991** [comment]
ci:forward

**Comment by victorg1991** [comment]
Niice 👏

**Pull request closed without merging**